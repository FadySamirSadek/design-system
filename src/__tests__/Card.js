import React from 'react'
import renderer from 'react-test-renderer'
import { Card, theme } from '..'

describe('Card', () => {
  test('renders small box shadow with default border', () => {
    const json = renderer.create(<Card boxShadowSize="sm" />).toJSON()
    expect(json).toMatchSnapshot()
    expect(json).toHaveStyleRule('box-shadow', theme.boxShadows[0])
    expect(json).toHaveStyleRule('border', `1px solid ${theme.colors.smoke}`)
    expect(json).toHaveStyleRule('border-radius', theme.radius)
  })
})
