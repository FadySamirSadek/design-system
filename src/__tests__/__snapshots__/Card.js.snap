// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders small box shadow 1`] = `
.c0 {
  box-shadow: 0 1px 4px 0 rgba(0,0,0,0.125);
  border-radius: 4px;
}

<div
  className="c0 "
/>
`;

exports[`Card renders with 1px border 1`] = `
.c0 {
  border: 1px solid #dde1e4;
  border-radius: 4px;
}

<div
  className="c0 "
/>
`;
